spring-jdbc/src/main/java/org/springframework/jdbc/datasource/init/ScriptUtils.java

```diff
@@ -309,11 +352,35 @@ private static String readScript(EncodedResource resource, @Nullable String comm
 	public static String readScript(LineNumberReader lineNumberReader, @Nullable String lineCommentPrefix,
 			@Nullable String separator, @Nullable String blockCommentEndDelimiter) throws IOException {
 
+		String[] lineCommentPrefixes = (lineCommentPrefix != null) ? new String[] { lineCommentPrefix } : null;
+		return readScript(lineNumberReader, lineCommentPrefixes, separator, blockCommentEndDelimiter);
+	}
+
+	/**
+	 * Read a script from the provided {@code LineNumberReader}, using the supplied
+	 * comment prefix and statement separator, and build a {@code String} containing
+	 * the lines.
+	 * <p>Lines <em>beginning</em> with the comment prefix are excluded from the
```

Comment:
```suggestion
	 * <p>Lines <em>beginning</em> with one of the comment prefixes are excluded from the
```

Reply:
Javadoc updates look great. Thanks for catching those.

Reply:
No problem. 

Does that mean you'd like me to click the "Commit suggestion" buttons, or are you planning on polishing the PR yourself?

Reply:
Since you're likely still sleeping, I'll just go ahead and do that. 😉